build-logic/build.gradle.kts

```diff
@@ -13,7 +13,6 @@ kotlin {
 }
 
 dependencies {
-    implementation(libs.gradlePlugin.dokka)
```

Comment:
I'm not sure why this dependency is here, but it's not used, and it broke Gradle dependency resolution.

Reply:
I do think it was there before, when we published Dokka generated documentation of Dokka in javadoc :)
I do think that we don't need it anymore even in version catalog, so feel free to remove it